internal/db/read_writer_test.go

```diff
@@ -3193,3 +3193,16 @@ func TestDb_oplogMsgsForItems(t *testing.T) {
 // 		})
 // 	}
 // }
+
+func TestDb_Now(t *testing.T) {
+	t.Parallel()
+	conn, _ := TestSetup(t, "postgres")
+	rw := New(conn)
+	ctx := context.Background()
+	now, err := rw.Now(ctx)
+	require.NoError(t, err)
+	// Check that it's within 1 second of now according to the system
+	// If this is flaky... just increase the limit 😬.
+	assert.True(t, now.Before(time.Now().Add(time.Second)))
+	assert.True(t, now.After(time.Now().Add(-time.Second)))
```

Comment:
Can we test calling `Now` multiple times within a single transaction and then multiple calls not within a transaction?
* When in the same transaction, `Now` should return identical `time.Time` values.
* When not in a transaction, `Now` should return different `time.Time` values.

Reply:
Thanks, done!